libs/shared-entity/src/dto/workspace_dto.rs

```diff
@@ -132,6 +132,7 @@ pub struct Page {
 pub struct CreatePageParams {
   pub parent_view_id: String,
   pub layout: ViewLayout,
+  pub name: Option<String>,
```

Comment:
should add set the serde(default) ? For example, old client connect to latest server might have serde issue

Reply:
This API is not being used at the moment, so backward compatibility is not an issue.